source/ItemBlock.hx

```diff
@@ -0,0 +1,55 @@
+package;
+
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import haxe.Timer;
+
+/**
+ * Item blocks that produce their items when hit from below
+ * @author TR Nale
+ */
+class ItemBlock extends Block
+{
+	public var containedItem:String; //placeholder var until we have an item class
+	public var isExpended:Bool; //whether the item has been removed yet
```

Comment:
Do you think there should be a separate class for the expended block, or at least some way to instantiate an already-expended item block?